mysql-test/main/mysqld--help.result

```diff
@@ -1503,9 +1503,11 @@ binlog-checksum CRC32
 binlog-commit-wait-count 0
 binlog-commit-wait-usec 100000
 binlog-direct-non-transactional-updates FALSE
+binlog-do-db (No default value)
 binlog-expire-logs-seconds 0
 binlog-file-cache-size 16384
 binlog-format MIXED
+binlog-ignore-db (No default value)
```

Comment:
Good job on fixing the help output! Now we not only support these two as variables, but also maintain good backward compatibility (ie. "--binlog-do-db-db1 --binlog-do-db=db2" from command line still works)!